### ISSUE TITLE
How to test the pretrained model you released? 

### ISSUE THREAD

**Issue by Zhiying-Li-dot** [caizhongang/SMPLer-X#74]
Hello, I face the problem. When I perform the test.py, I can't run. So I set `    
config_path = '/data/zhiyingli/SMPLer-X/main/config/config_smpler_x_h32.py'
ckpt_path = "/data/zhiyingli/SMPLer-X/pretrained_models/smpler_x_h32_correct.pth.tar"
`. But the result for PW3D can't align the result in your paper. How to resolve the problem?
![38c5f2989f0c575ddb569b985520f5d](https://github.com/user-attachments/assets/5c40c179-e0f4-4b05-8ddb-c2e46bb38ed2)

**Comment by piumalnipun9** [comment]
Hi do we need to dowmload all models specified under pretrained model to do a inference. Or just smpler-x model is enough. Thank you.

**Comment by Zhiying-Li-dot** [comment]
> Hi do we need to dowmload all models specified under pretrained model to do a inference. Or just smpler-x model is enough. Thank you.

Yes, you need to download all the files according to the directory structure outlined in the README before you can proceed with the inference.
